UsersGitHub/UsersGitHub/View/LoginPage.xaml.cs

```diff
@@ -3,12 +3,13 @@
 
 namespace UsersGitHub.View
 {
-	[XamlCompilation(XamlCompilationOptions.Compile)]
-	public partial class LoginPage
-	{
-		public LoginPage()
-		{
-			InitializeComponent();
-		}
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class LoginPage
+    {
+        public LoginPage()
+        {
+            InitializeComponent();
+            BindingContext = new LoginPageViewModel(() => DisplayAlert("Error", "This name doesn't exist", "OK"));
```

Comment:
you are passing same alert to another page
may be it's better to extract it to a separate helper?